goaviatrix/edge_neo_proxy_profile.go

```diff
@@ -0,0 +1,88 @@
+package goaviatrix
```

Comment:
Since this a new file, lets not use the term `neo` lets call this edge_platform_proxy_profile (or something like that)